readme.md

```diff
@@ -44,6 +44,12 @@ Returns an array of objects in the format `{ pattern: string, opts: Object }`, w
 
 Note that you should avoid running the same tasks multiple times as they contain a file system cache. Instead, run this method each time to ensure file system changes are taken into consideration.
 
+### globby.hasMagic(pattern, [options]);
```

Comment:
Personally, I'm unwilling to keep documentation in sync with `node-globs`. Furthermore, it smells like plagiarism. As we don't modify behavior, I would suggest:
> ### globby.hasMagic
> See `node-glob` [hasMagic](https://github.com/isaacs/node-glob#globhasmagicpattern-options).